internal/pkg/plugin/reposcaffolding/github/golang/golang.go

```diff
@@ -18,6 +18,8 @@ const (
 	DefaultWorkPath      = ".github-repo-scaffolding-golang"
 	DefaultTemplateRepo  = "dtm-scaffolding-golang"
 	DefaultTemplateOwner = "merico-dev"
+	TransBranch          = "init-with-devstream"
```

Comment:
"Trans" as a prefix can mean many things, transform, transaction, translate, etc. I think we can use a longer full name here to be perfectly clear.

Reply:
ok

Reply:
TransitBranch is better here